### PR TITLE
b-wav mime type support added

### PR DESCRIPTION
**Pull request by skywojciechowskim**
Summary: b-wav mime type support added
Type: Feature
Test Plan: UT/CT/YT Conformance, YT Codec
Jira: RIALTO-592

### CODE REVIEW

File: media/server/gstplayer/source/tasks/generic/AttachSource.cpp [hunk 1]
```diff
@@ -127,6 +128,10 @@ class MediaSourceAudioCapsBuilder : public MediaSourceCapsBuilder
         {
             addMpegVersionToCaps(caps);
         }
+        if (mimeType == "audio/b-wav")
```
Comment (skyproudmanp) [inline media/server/gstplayer/source/tasks/generic/AttachSource.cpp:131]: "else if" could be used here

File: media/server/gstplayer/source/tasks/generic/AttachSource.cpp [hunk 2]
```diff
@@ -140,6 +145,7 @@ class MediaSourceAudioCapsBuilder : public MediaSourceCapsBuilder
         {
             caps = m_gstWrapper->gstCapsNewSimple("audio/x-opus", "channel-mapping-family", G_TYPE_INT, 0, nullptr);
             addSampleRateAndChannelsToCaps(caps);
+            firebolt::rialto::AudioConfig audioConfig = m_attachedAudioSource.getAudioConfig();
```
Comment (skyproudmanp) [inline media/server/gstplayer/source/tasks/generic/AttachSource.cpp:148]: Unused variable audioConfig could be removed

File: media/server/gstplayer/source/tasks/generic/AttachSource.cpp (hunk 1 above)
Comment (skywojciechowskim) [reply media/server/gstplayer/source/tasks/generic/AttachSource.cpp]: changed

File: media/server/gstplayer/source/tasks/generic/AttachSource.cpp (hunk 2 above)
Comment (skywojciechowskim) [reply media/server/gstplayer/source/tasks/generic/AttachSource.cpp]: removed

**Review by skyproudmanp — APPROVED** [review]

**Review by skywilliamsonl — APPROVED** [review]

**Review by skywilliamsonl — APPROVED** [review]

**Review by skyproudmanp — APPROVED** [review]

**Pull request merged**